.appveyor.yml

```diff
@@ -23,13 +23,14 @@ environment:
     # In theory we could use a single CONDA_INSTALL_LOCN because we construct
     # the envs anyway. But using one for the right python version hopefully
     # making things faster due to package caching.
-    - PYTHON_VERSION: "3.6"
-      CONDA_INSTALL_LOCN: "C:\\Miniconda36-x64"
+    - PYTHON_VERSION: "3.7"
+      CONDA_INSTALL_LOCN: "C:\\Miniconda37-x64"
       TEST_ALL: "no"
       EXTRAREQS: "-r requirements/testing/travis_extra.txt"
-    - PYTHON_VERSION: "3.7"
+    - PYTHON_VERSION: "3.8"
       CONDA_INSTALL_LOCN: "C:\\Miniconda37-x64"
```

Comment:
```suggestion
      CONDA_INSTALL_LOCN: "C:\\Miniconda38-x64"
```

Reply:
This doesn't work unless we update the windows image we are using.